docs/docs/modeling.md

```diff
@@ -114,7 +114,25 @@ classDiagram
 
 ## AssignmentSheet - 分担票
 
-## TenantsWork -
+## TenantsWork - 割当
+
+TenantsWork refers to the set of tasks assigned(AssignedCategory) to each tenant.
+
+### Tenant IDs
+
+The TenantsWork contains Tenants. The TenantsWork class has Tenant ID.
+
+Constraints:
+
+- The TenantsWork cannot exist without any Tenants.
```

Comment:
There is always only one `TenantId` that exists in the `TenantWork` as the `TenantsWork` is created per `Tenant`.